src/Applications/beta/examples/Set-MgBetaApplicationConnectorGroupByRef.md

```diff
@@ -1,10 +1,15 @@
-### Example 1: Using the Set-MgBetaApplicationConnectorGroupByRef Cmdlet
+### Example 1: Code snippet
+
 ```powershell
+
 Import-Module Microsoft.Graph.Beta.Applications
+
 $params = @{
-	"@odata.id" = "https://graph.microsoft.com/onPremisesPublishingProfiles/applicationproxy/connectorGroups/{id}"
+	"@odata.id" = "https://graph.microsoft.com/beta/onPremisesPublishingProfiles/applicationproxy/connectorGroups/{id}"
 }
+
 Set-MgBetaApplicationConnectorGroupByRef -ApplicationId $applicationId -BodyParameter $params
+
 ```
 This example shows how to use the Set-MgBetaApplicationConnectorGroupByRef Cmdlet.
-To learn about permissions for this resource, see the [permissions reference](/graph/permissions-reference).
+
```

Comment:
Why did you remove the notice on permissions?

Reply:
That bit is general and not specific to the path related to this command. We handled that issue by including the exact permissions table during weekly reference docs refresh. Have a look at this https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.beta.applications/set-mgbetaapplicationconnectorgroupbyref?view=graph-powershell-beta

<img width="464" alt="image" src="https://github.com/microsoftgraph/msgraph-sdk-powershell/assets/10947120/cbb32938-942e-4c43-ae50-da6c6865dd60">